src/Pages/VideoPage.js

```diff
@@ -0,0 +1,139 @@
+import { useParams } from "react-router-dom";
+// import YouTube from "react-youtube";
+import { VideoList } from "./VideoList";
+import { ActionTypes, useData, useAuth } from "../Context";
+import { YoutubeVideoDisplay, PlaylistModal } from "../Components";
+import { useEffect, useState } from "react";
+export const Video = () => {
+  return (
+    <div>
+      <VideoDisplay />
+      <VideoList />
+    </div>
+  );
+};
+
+const VideoDisplay = () => {
+  const { videoId } = useParams();
+  const { state, dispatch, showSnackBar, setSnackBarContent } = useData();
+  const {
+    authState: { isLoggedIn },
+  } = useAuth();
+  const [displayModal, setDisplayModal] = useState("none");
+  const video = state.videos.find((item) => item.id === videoId);
+  useEffect(() => {
+    document.title = video.description;
+  }, [video.description]);
+
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, [videoId]);
+
+  return (
+    <div>
+      <div className="iframe-container">
+        <YoutubeVideoDisplay youtubeId={videoId} />
+      </div>
+      <PlaylistModal
+        displayState={displayModal}
+        setDisplayState={setDisplayModal}
+      />
+      <div>
+        <div style={{ display: "flex" }}>
+          <div className="avatar-container-sm avatar-padding">
+            <img
+              className="avatar avatar-padding"
+              src={video.avatarSrc}
+              alt={video.uploadedBy}
+            />
+          </div>
+          <p>{video.description}</p>
+        </div>
+        <div
+          style={{
+            display: "flex",
+          }}
+        >
+          {inLikedVideos({ id: video.id, likedVideos: state.liked }) ? (
+            <button
+              className="icon-button button-style"
+              onClick={() => {
+                setSnackBarContent(`Removed from Liked videos`);
+                showSnackBar();
+                dispatch({
+                  type: ActionTypes.REMOVE_FROM_LIKED,
+                  payload: video,
+                });
+              }}
+            >
+              <i class="fas fa-thumbs-up"></i>
+            </button>
+          ) : (
+            <button
+              className="icon-button button-style"
+              onClick={() => {
+                if (isLoggedIn) {
```

Comment:
extract this kind of click handler(having lines >= 3) to a separate function which will make the code cleaner